src/test/java/jimmy/mcgymmy/model/food/CarbohydrateTest.java

```diff
@@ -8,9 +8,14 @@
 
 public class CarbohydrateTest {
 
+    @Test
+    public void newDefault_works() {
+        // If it doesnt, the thrown runtime exception will break this test.
+        Carbohydrate.newDefault();
+    }
```

Comment:
Should this be written slightly differently? Maybe `Assertions.assertTrue(Carbohydrate.isValid(Carbohydrate.newDefault()));` instead? 

Similar instances for other Macronutrient tests

Reply:
This is just checking if the function works i.e. a default carbohydrate can be created properly. If there are any problems a runtime exception/assertion will be thrown and this test will fail.

the `isValid` method is for checking user input (strings).

Reply:
Okay sure